routes/error-tracker.js

```diff
@@ -49,6 +47,39 @@ function ignoreMessageOrException(message, exception) {
 }
 
 /**
+ * @param {string} exception
+ * @return {string} standardized exception
+ */
+function stackTraceConversion(exception) {
+  const location = '([^ \\n]+):(\\d+):(\\d+)';
+  const chromeStackTraceRegex = new RegExp(`^\\s*at (.+ )?(?:${location}|\\(${location}\\))$`, 'gm');
+  const mozillaSafariStackTraceRegex = /^([^@\n]*)@(.+):(\d+):(\d+)$/gm;
+  let validException = '';
+  let match;
+  let validExceptions = [];
+  if (chromeStackTraceRegex.test(exception)) {
+    while ((match = chromeStackTraceRegex.exec(exception))) {
+      validExceptions.push(match[0]);
+    }
+  } else {
+    let otherMatch;
+    while (otherMatch = mozillaSafariStackTraceRegex.exec(exception)) {
+      validExceptions.push(safariOrMozillaToChrome(otherMatch));
+    }
+    validException = validExceptions.join('\n');
+  }
+  return validException;
+}
+
+/**
+ * @param {string} match
+ * @return {string} chromeLikeException
+ */
+function safariOrMozillaToChrome(match) {
+  return ` at ${match[1]} ${match[2]}:${match[3]}:${match[4]}`;
+}
```

Comment:
This function doesn't really make sense outside of the scope of `stackTraceConversion()`. I'd inline this instead:

```js
    while (otherMatch = mozillaSafariStackTraceRegex.exec(exception)) {
       const chromeLikeException = ` at ${otherMatch[1]} ${otherMatch[2]}:${otherMatch[3]}:${otherMatch[4]}`;
       validExceptions.push(chromeLikeException);
     }
```